README.md

```diff
@@ -9,247 +9,128 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
-    make && sudo make install ; cd ..
-
-You have successfully built and installed the AWS Encryption SDK for C.
-
-### Building on Ubuntu
-
-These instructions have been tested on brand new Ubuntu EC2 instances. You should also
-be able to build on Ubuntu operating systems that are not in EC2, but you will need to
-manually configure AWS credentials if you are using KMS. Start in the directory where
-you want to do your build.
+1. OpenSSL 1.0.2 or newer, or 1.1.0 or newer
+1. CMake 3.9 or newer
+1. C/C++ compiler
+1. aws-c-common, typically bundled with the AWS SDK for C++
+1. The AWS SDK for C++ version 1.9.35 or newer
 
-#### Ubuntu: (Option 1) C and C++ build dependencies
+The AWS SDK for C++ and the AWS Encryption SDK for C share dependencies on OpenSSL, aws-c-common, and CMake, and 
+the AWS SDK for C++ has some additional dependencies and prerequisites of its own. See [AWS SDK for
+C++: Getting Started](https://github.com/aws/aws-sdk-cpp#getting-started).
 
-    sudo apt-get update
-    sudo apt-get install -y libssl-dev cmake g++ libcurl4-openssl-dev zlib1g-dev
-
-Both aws-sdk-cpp and aws-c-common are required, but the installation of aws-sdk-cpp will install
-aws-c-common for you.
-
-Do a KMS-only build of the AWS SDK for C++. If you want to use the AWS SDK for C++ for
-other AWS services, you can omit the `-DBUILD_ONLY="kms"` argument, but the build will take much longer.
-
-    git clone https://github.com/aws/aws-sdk-cpp.git
-    mkdir build-aws-sdk-cpp && cd build-aws-sdk-cpp
-    cmake -DBUILD_SHARED_LIBS=ON -DBUILD_ONLY="kms" -DENABLE_UNITY_BUILD=ON ../aws-sdk-cpp
-    make && sudo make install ; cd ..
+You need to compile both the AWS Encryption SDK for C and its dependencies as either all
+shared or all static libraries. These instructions will use all static library builds. Static library
+builds are the default for the AWS SDK for C++ and aws-c-common.
 
-Now skip to the "Ubuntu: Build and install the AWS Encryption SDK for C" section below.
+If you would like to build shared libraries, you will need to supply the `-DBUILD_SHARED_LIBS=ON` flag to build
+aws-c-common, the AWS SDK for C++, and the AWS Encryption SDK.
 
-#### Ubuntu: (Option 2) C only build dependencies
+### Linux Build Recipe
 
-    sudo apt-get update
-    sudo apt-get install -y libssl-dev cmake gcc
+First you will need to build the AWS SDK for C++. That will install the shared dependencies.
 
-Now build and install aws-c-common:
+If you only need AWS SDK for C++ to use the AWS Encryption SDK, you have the option to build only the AWS KMS SDK.
+Add the `-DBUILD_ONLY="kms"` flag to `cmake` in the instructions provided.
 
-    git clone https://github.com/awslabs/aws-c-common.git
-    mkdir build-aws-c-common && cd build-aws-c-common
-    cmake -DBUILD_SHARED_LIBS=ON ../aws-c-common
-    make && sudo make install ; cd ..
+[Follow the AWS SDK for C++ build instructions](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/setup-linux.html).
```

Comment:
Delete. Moved this into the previous sentence.

Reply:
I put the instructions in this order with the intent that I state the required step (first, build the SDK); then I explain what modifications the customer must make to the instructions ("if you only need..."); then I finally link to the directions after clarifying what needs to change.

My concern about putting the instructions link first/combined with the previous sentence is that the customer will correctly click the link and go follow those instructions without realizing that our instructions have a couple of notes that apply to those steps.

Does that make sense? Should I still change this, or change it in some different way?

Reply:
That's fine.